### PR TITLE
Aleksandr Semenov. Github Task.

### PR DESCRIPTION
**Pull request by IAMDOKTOR**
@tanvirsinghahuja @StandingMike

Hey, guys

Please take a review on my commit for tasks 1 & 2.

### CODE REVIEW

**Comment by StandingMike** [comment]
@alex-andrero

Looks good to me!

I'll wait for @tanvirsinghahuja  to review before merging

**Review by StandingMike — APPROVED** [review]

**Review by tanvirsinghahuja — APPROVED** [review]

**Pull request merged**
